src/rgw/rgw_data_sync.h

```diff
@@ -285,15 +287,14 @@ class RGWRemoteDataLog : public RGWCoroutinesManager {
   bool initialized;
 
 public:
-  RGWRemoteDataLog(const DoutPrefixProvider *dpp, RGWRados *_store,
-                   RGWAsyncRadosProcessor *async_rados,
+  RGWRemoteDataLog(const DoutPrefixProvider *dpp, RGWRados *_store, RGWAsyncRadosProcessor *async_rados,
                    rgw::BucketChangeObserver *observer)
     : RGWCoroutinesManager(_store->ctx(), _store->get_cr_registry()),
       dpp(dpp), store(_store), async_rados(async_rados), observer(observer),
       http_manager(store->ctx(), completion_mgr),
       lock("RGWRemoteDataLog::lock"), data_sync_cr(NULL),
       initialized(false) {}
-  int init(const string& _source_zone, RGWRESTConn *_conn, RGWSyncErrorLogger *_error_logger,
+  int init(const DoutPrefixProvider *_dpp, const string& _source_zone, RGWRESTConn *_conn, RGWSyncErrorLogger *_error_logger,
```

Comment:
`RGWRemoteDataLog` already has a `dpp` from its constructor, so we shouldn't need to pass another into `init()`